### PR TITLE
Javascript project popupmenu

### PR DESCRIPTION
**Pull request by paulshaun-kdtk**
## Changes

- Added dynamic JS pop-up that relies on screen width and @media-queries
- Rebased index.html file to accommodate dynamic changes
- Styled pop-up according to Figma template

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,372 +1,216 @@
...
           <p>
-            Hello I’m a software developer! I can help you build a product,
-            feature or website Look through some of my work and experience! If
-            you like what you see and have a project you need coded, don’t
-            hesitate to contact me.
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required has been the industry's standard dummy text ever
+            since the 1500s, when an unknown printer took a standard dummy
+            text.
           </p>
```
Comment (emmiiorji) [inline index.html:112]: Well done for loading some of the projects from JavaScript. As specified in the requirements, kindly ensure that all projects are loaded the same way.

File: index.js [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+      <article>
+        <p>${projectDetails[elementIndex].details}</p>
+        <div>
+          <button class="btn" type="button"><a href='${projectDetails[elementIndex].live}'>See Live</a></button>
+          <button class="btn" type="button"><a href='${projectDetails[elementIndex]['github link']}'>See Source</a></button>
+        </div>
+      </article>
+    </section>
+    `;
+    popUp.appendChild(modal);
```
Comment (emmiiorji) [inline index.js:168]: You seem to have the same content for every project. Kindly ensure that the modal is populated with data from the project array and will correspond with the clicked project.

image

File: index.js [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+    </ul>
+    <section>
+      <div>
+        <img src=${projectDetails[elementIndex].img} alt="An image of the project">
+      </div>
+      <article>
+        <p>${projectDetails[elementIndex].details}</p>
+        <div>
+          <button class="btn" type="button"><a href='${projectDetails[elementIndex].live}'>See Live</a></button>
+          <button class="btn" type="button"><a href='${projectDetails[elementIndex]['github link']}'>See Source</a></button>
```
Comment (emmiiorji) [inline index.js:163]: Kindly ensure that the buttons contain the icons as given by the design.

Your design:
image

Figma design:
image

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @paulshaun-kdtk and Team, 
# Status: Changes Required ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<div align='center'>

</div>

## Good Highlights✔️

- Projects are rendered from JavaScript
- The close button works as expected


## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by paulshaun-kdtk** [comment]
@emmiiorji  please clarify on your first comment

**Comment by emmiiorji** [comment]
Hi @paulshaun-kdtk, you seem to still have the highlighted project still rendered from html.

**Comment by paulshaun-kdtk** [comment]
@emmiiorji 
are you up for a slack huddle so you can clarify a bit more?

**Comment by paulshaun-kdtk** [comment]
Oh nevermind... i get what you mean sorry about that.

**Comment by emmiiorji** [comment]
Okay, great.

File: index.html [hunk 4]
```diff
@@ -1,372 +1,218 @@
...
-      <section class="about-section">
-        <article class="about-details">
-          <h3>About me</h3>
+          <img src="./assets/images/mp-img.svg" alt="image of multi-post stories" />
+          <br />
+        <br />
+        </div>
+        
+        <article class="mp-article">
+          <h2 class="heading">Multi-Post Stories</h2>
```
Comment (V-Blaze) [inline index.html:106]: - [OPTIONAL] Following the best practices, it is not advisable to leave unused code as a comment on your codebase, it makes your code base look unprofessional. I'll suggest you remove this. :smile: :+1:

**Review by V-Blaze — APPROVED** [review]
Hi @paulshaun-kdtk  , 

# Approved 🟢 🟢 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy

## Highlights

- Project section generated dynamically💯
-  Popup works as expected👍🟢 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**